catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrMetacardClientImpl.java

```diff
@@ -125,6 +129,18 @@
   private static final Supplier<Boolean> ZERO_PAGESIZE_COMPATIBILTY =
       () -> Boolean.valueOf(System.getProperty(ZERO_PAGESIZE_COMPATIBILITY_PROPERTY));
 
+  @VisibleForTesting
+  static final String SOLR_COMMIT_NRT_METACARDTYPES = "solr.commit.nrt.metacardTypes";
+
+  private final Set<String> commitNrtMetacardType =
+      Sets.newHashSet(accessProperty(SOLR_COMMIT_NRT_METACARDTYPES, "").split(","));
```

Comment:
✏️ `split("\s*,\s*")`

Reply:
@paouelle ❓ Would it be safer to bound it to horizontal whitespace only? `split("\h*, \h*")`

Reply:
safer why? do we care about making the distinction or just trimming?